plaso/storage/interface.py

```diff
@@ -698,6 +759,27 @@ def __init__(self, path):
     self._path = path
     self._storage_file = None
 
+  @property
+  def format_version(self):
+    """int: format version"""
```

Comment:
please add something in line of `or None if not set`

Reply:
Done